src/nginxconfig/i18n/en/templates/domain_sections/reverse_proxy.js

```diff
@@ -31,4 +31,5 @@ export default {
     reverseProxyCannotBeEnabledWithPhp: `${common.reverseProxy} cannot be enabled whilst ${common.php} is enabled.`,
     reverseProxyCannotBeEnabledWithPython: `${common.reverseProxy} cannot be enabled whilst ${common.python} is enabled.`,
     enableReverseProxy: `${common.enable} ${common.reverseProxyLower}`,
+    proxyHeaderOverride: 'Proxy header override',
```

Comment:
```suggestion
    proxyHeaderOverride: 'Proxy Host header',
```
I think we should call out the header this affects here, for clarity (it also isn't really an override given the default is `$host`).